### PR TITLE
Mark notifications as read when opening

### PR DESCRIPTION
**Pull request by phildarnowsky**
Card: https://app.asana.com/0/1148853526253426/1192202587402595

Note: this is based on https://github.com/mbta/skate/pull/966

### CODE REVIEW

File: assets/src/components/notificationDrawer.tsx [hunk 1]
```diff
@@ -43,79 +46,36 @@ const TitleBar = () => {
 }
 
 const Content = () => {
-  const { notifications } = useContext(NotificationsContext)
+  const { dispatch, notifications } = useContext(NotificationsContext)
   const currentTime = useCurrentTime()
 
   const [, stateDispatch] = useContext(StateDispatchContext)
 
+  const notificationsDispatch = useContext(NotificationsContext).dispatch
```
Comment (lemald) [inline assets/src/components/notificationDrawer.tsx:54]: Am I missing something or is `notificationsDispatch` the same value as `dispatch` (but distinct from `stateDispatch`)?

Comment (phildarnowsky) [reply assets/src/components/notificationDrawer.tsx:54]: whoops...so it is.

**Review by lemald — APPROVED** [review]
Looks good to me. There's the question of what to do about the inactive notification modal when opened from the drawer rather than from the ladder page, but I think that's orthogonal and can be addressed in a different PR.

**Comment by phildarnowsky** [comment]
@lemald anything else on this one?

**Pull request merged**